src/Tribe/Admin/Notices.php

```diff
@@ -30,9 +30,12 @@ public function maybe_display_notices() {
 			return;
 		}
 
-		$this->maybe_display_plus_commerce_notice();
+		include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
```

Comment:
:warning: "include_once" is a statement not a function; no parentheses are required

<sub>hash: 0a933503527cba3e16625b5650ad8052</sub>

Reply:
remove the parentheses here.